Tasks/AzureWebPowerShellDeployment/Strings/resources.resjson/zh-TW/resources.resjson

```diff
@@ -23,5 +23,7 @@
   "loc.messages.CannotupdatedeploymentstatusuniquedeploymentIdcannotberetrieved": "無法更新部署狀態，無法擷取獨一的 deploymentId",
   "loc.messages.Updatingdeploymenthistoryfordeployment0": "正在更新部署 {0} 的部署記錄",
   "loc.messages.Nofileswerefoundtodeploywithsearchpattern0": "找不到任何檔案可用搜尋模式 {0} 部署",
-  "loc.messages.Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "使用搜尋模式 {0} 找到一個以上要部署的檔案。但僅允許一個。"
+  "loc.messages.Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "使用搜尋模式 {0} 找到一個以上要部署的檔案。但僅允許一個。",
+  "loc.messages.FailedtodeployWebsiteError0": "無法部署網站。錯誤: {0}",
+  "loc.messages.DeprecatedWarning": "'Azure App Service: Classic' task will be deprecated soon. 'Azure App Service Deploy' task will replace 'Azure App Service: Classic' task and the recommendation is to migrate your Build or Release process to use the 'Azure App Service Deploy' task. Refer https://go.microsoft.com/fwlink/?LinkID=613750 for more details."
```

Comment:
yes. the fact that the english string is getting generated is a good sign. it should mean the ball is in localizers court.

i just ran through handoff steps. so i'll check back next week to see if more strings are back. today is the last handoff. handback will continue next week, and likely wont meet the bar after that once we enter escrow mode.